src/ColorVectorSpace.jl

```diff
@@ -8,7 +8,7 @@ import Base: ==, +, -, *, /, .+, .-, .*, ./, ^, <, ~
 import Base: abs, abs2, clamp, convert, copy, div, eps, isfinite, isinf,
     isnan, isless, length, mapreduce, norm, one, promote_array_type, promote_rule, zero,
     trunc, floor, round, ceil, bswap,
-    mod, rem, atan2, hypot, max, min, varm, real
+    mod, rem, atan2, hypot, max, min, histrange
```

Comment:
This was by mistake. I did not merge the conflicts correctly. I did not remember to run tests as I felt I had not added anything, but I forgot someone else would have! :P 

I'll resubmit this after correcting the conflict.